src/data/minors/animal-sci.ts

```diff
@@ -0,0 +1,79 @@
+import { CollegeOrMajorRequirement } from '../../requirements/types';
+import { includesWithSingleRequirement } from '../../requirements/checkers';
+import { AdvisorGroup } from '../../tools/advisors/types';
+
+const animalSciRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Categories',
+    description:
+      'The minor is satisfied by completing at least 15 credit hours of Animal Science courses.',
+    source:
+      'https://cals.cornell.edu/animal-science/degrees-programs/undergraduate-studies/minor-requirements',
+    checker: includesWithSingleRequirement(
```

Comment:
This can be fulfilled by any animal science class right? We should use the `ifCodeMatch` checker to determine whether a class is an animal science course--check out the CS major file for an example.

Reply:
I think you would also have to check if the course is "Special Studies." Something like "if the code doesn't match special studies and matches animal science."

Reply:
Yeah, you can probably do `(ifCodeMatch(ANSC 1****) || ifCodeMatch(ANSC 2****) || ifCodeMatch(ANSC 3****) || ifCodeMatch(ANSC 4****)) && !ifCodeMatches(ANSC 49**)`

where, ANSC 49** are special studies courses.

Reply:
I think that's true actually. I'll also remove any 5000+ courses as well.